### DISCUSSION TITLE
Access historical alert more than last 7 days

### DISCUSSION THREAD

**Discussion by clearlqy99** [weather-gov/api#638] [Q&A]
I'm interested in performing some analysis on historical weather alert data. Based on API documentation, the https://api.weather.gov/alerts endpoint provides historical alerts for the last 7 days. Is there any possible way to access historical data for a longer time span?

Thank you.

**Comment by akrherz** [comment]
I do not believe the NWS/NCEI has such archives of these alerts.  My work has a couple of resources that aren't exactly APIs, but may help your analysis.

- per UTC date/hour archives of the raw [CAP text messages](https://mtarchive.geol.iastate.edu/2023/04/27/text/cap/)
- GIS/KML/Excel [downloads](https://mesonet.agron.iastate.edu/request/gis/watchwarn.phtml) of VTEC enabled events
- Varios [adhoc JSON services](https://mesonet.agron.iastate.edu/json/) that may be helpful.

Please let me know akrherz@iastate.edu of any questions / help you need with what you are hoping to do.
